### PR TITLE
Ft index html

### PR DESCRIPTION
**Pull request by kbjude**
#### What does this PR do?
Create a youtube clone page.

#### Description of Task to be completed?
A clone page with an embedded youtube video 

#### Screenshots or gifs (if appropriate)
Screen Shot 2019-11-27 at 22 57 00

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  <body>
+    <header class="nav-bar">
+      <img src="images/download.png" class="logo" />
+      <form class="search-field">
+        <input type="text" class="search-input" placeholder="Search" />
+        <button type="submit" class="search-btn">
+          <i class="fa fa-search grey-icon"></i>
+        </button>
+      </form>
+      <i class="fa fa-upload upload-icon grey-icon" aria-hidden="true"></i>
```
Comment (pelzolga123) [inline index.html:25]: Navigation bar should have at least three icons on the rightmost part of the navbar.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Navigation bar should have some changes and Readme file needs to be more descriptive.

**Comment by pelzolga123** [comment]
## Hi Team 
### Your project looks :+1: amazing!

But there are few things that need to be changed.
* [x] Project should have at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar.
 Example:
image

* [x] Please make more descriptive Readme file. It will make your project look a lot more professional. [See an example here](https://github.com/pelzolga123/Best-README-Template)

Optional: you can make aside section smaller, so that project will look more like an original.

### Please kindly Fix the above issues and send another code review request.
If you have any questions or need help, you can message me on slack. *Happy coding*

File: index.html (hunk 1 above)
Comment (tunedev) [reply index.html:25]: Thanks for the feedback, it has been resolved. kindly confirm if it satisfies your request @pelzolga123

**Comment by tunedev** [comment]
> ## Hi Team
> ### Your project looks 👍 amazing!
> But there are few things that need to be changed.
> 
> * [x]  Project should have at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar.
>   Example:
>   image
> * [x]  Please make more descriptive Readme file. It will make your project look a lot more professional. [See an example here](https://github.com/pelzolga123/Best-README-Template)
> 
> Optional: you can make aside section smaller, so that project will look more like an original.
> 
> ### Please kindly Fix the above issues and send another code review request.
> If you have any questions or need help, you can message me on slack. _Happy coding_

Thanks for the feedback, kindly confirm the resolution meets your request.

**Pull request merged**